apps/site/lib/site_web/templates/partial/_expandable_block.html.eex

```diff
@@ -0,0 +1,34 @@
+
+<% container_id = "container-#{@id}" %>
+<% header_id = "header-#{@id}" %>
+<% panel_id = "panel-#{@id}" %>
+
+<div id="<%= container_id %>">
+  <h2 class="c-expandable-block__header">
+    <a class="c-expandable-block__link"
+      href="#<%= panel_id %>"
+      tabIndex="0"
+      id="<%= header_id %>"
+      aria-expanded="false"
+      aria-controls="<%= panel_id %>"
+      data-toggle="collapse">
+
+      <%= if @header.iconSvgText do %>
+        <%= content_tag(:span, svg(@header.iconSvgText), class: "notranslate c-expandable-block__header-icon", aria: [hidden: "true"]) %>
+      <% end %>
+      <div class="c-expandable-block__header-text">
+        <%= @header.text %>
+        <span class="c-expandable-block-caret"></span>
+      </div>
+    </a>
+  </h2>
+  <div
+    class="c-expandable-block__panel collapse<%= if @show_expanded do %> expand<% end %>"
```

Comment:
I tried that at first and the corresponding blocks are expanded initially but then that messes up with the carets :/
<img width="264" alt="image" src="https://user-images.githubusercontent.com/61979382/92589585-c0991580-f268-11ea-8c06-2b9ff17b7ad7.png">

That is why I took the jQuery approach to simulate clicks when the page loads. Do you think this can be achieved only with CSS?

Reply:
I think the caret part would be addressed by setting the aria-expanded to match the @show_expanded state on load.

Was going to write this in the @show_expanded comment but basically: the aria-expanded state should always reflect whether the block is collapsed or not. Then the caret will be pointed appropriately (since your CSS uses aria-expanded state to choose which caret to show). This is handled on click fine, but if it's set to false on load when @show_expanded happens to be true, then there's an inconsistency at the start.